index.html

```diff
@@ -88,6 +92,132 @@
+				<div class="document-card-container">
+					<ts-document-card
+						name="Invoice #344234"
+						mobile-description="Tradeshift SRL | 6 participants"
+						description="Tradeshift SRL"
+					></ts-document-card>
+					<ts-document-card
+						name="looooooooooooooooooooooooooooooooooooooooong teeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeext"
+						description="Tradeshift looooooooooooooooooooooooooooooooooooooooong"
+						mobile-description="Mobile looooooooooooooooooooooooooooooooooooooooong"
+					></ts-document-card>
+				</div>
+			</article>
+			<article>
+				<h2>Table</h2>
+				<ts-table id="ts-table-demo"></ts-table>
+				<script>
+					function handleRowClick(e) {
+						console.log('handleRowClick', e);
+					}
+					const tsTable = document.getElementById('ts-table-demo');
+					tsTable.addEventListener('row-click', handleRowClick);
+					tsTable.cols = [
+						{
+							type: 'document',
+							property: 'context',
+							size: 'large',
+							value: 'Context'
+						},
+						{ visibility: 'desktop', type: 'status', property: 'type', size: 'small', value: 'Type', display: 'left' },
+						{ visibility: 'desktop', property: 'participants', size: 'small', value: 'Participants' },
+						{ visibility: 'desktop', property: 'newActivity', value: 'New Activity' },
+						{
+							visibility: 'desktop',
+							property: 'lastActivity',
+							value: 'Last Activity',
+							size: 'medium',
+							display: 'right'
+						},
+						{
+							visibility: 'mobile',
+							property: 'mobileLastActivity',
+							size: 'small',
+							display: 'right',
+							renderFunction(item) {
+								const div = document.createElement('div');
+								div.style.padding = '10px';
+								div.style.whiteSpace = 'nowrap';
+								if (item.unread && item.unread > 0) {
+									const child = document.createElement('div');
+									child.innerText = item.unread;
+									child.style.backgroundColor = '#01b0ff';
+									child.style.borderRadius = '3px';
+									child.style.color = 'white';
+									child.style.padding = '5px';
+									child.style.fontSize = '11px';
+									child.style.fontWeight = '600';
+									div.style.display = 'flex';
+									div.style.alignItems = 'center';
+									div.appendChild(child);
+								} else {
+									div.innerText = item.lastActivity;
+								}
+								return div;
+							}
+						}
+					];
+
+					tsTable.data = [
+						{
+							content: {
+								context: {
+									name: 'Invoice #32131',
+									description: 'Tradeshift',
+									mobileDescription: 'Tradeshift | 5 participants'
+								},
+								type: { status: 'ok', text: 'Internal' },
+								participants: 100,
+								newActivity:
+									'Every little bunny has a habit that is funny. It doesnt matter where he goes he always wrinkles up his nose.',
+								lastActivity: 'Yesterday, 15:01',
+								mobileLastActivity: { unread: 10, lastActivity: 'Yesterday, 15:01' }
+							},
+							isSelected: true
```

Comment:
In my opinion, data about selected rows should be separated, not related to the table cells data. It also will simplify the data structure, eliminating the one level of nesting (you can remove the `content` property and provide only data, required for table render).
To expose it I suggest to provide an API to select particular row/rows and get a list of selected rows.
E.g.:
`selectRow(<Array of rows ids>)`
`getSelectedRows() <Array of selected rows>`

Reply:
done

Reply:
but I've done it by sending an array of ids that should be selected